SPEC.md

```diff
@@ -314,22 +316,22 @@ Retrievable at `http://$AC_METADATA_URL/acMetadata/v1/container`
 
 | Entry       | Description |
 |-------------|-------------|
-|annotations/ | Top level annotations from container runtime manifest.|
-|manifest     | The container manifest JSON |
-|uid          | The unique execution container uid.|
+|annotations/ | Top level annotations from container runtime manifest. |
+|manifest     | The container manifest JSON. |
+|uid          | The unique execution container uid. |
```

Comment:
Maybe this would be obvious from reading `rkt`, but: what's responsible for generating this uid, and is it already used for something specifically, or just sort of reserved for future use?